docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/DockerClientExecutor.java

```diff
@@ -782,6 +784,15 @@ private void configureBuildCommand(Map<String, Object> params, BuildImageCmd bui
         if (params.containsKey(DOCKERFILE_NAME)) {
             buildImageCmd.withDockerfile(new File((String) params.get(DOCKERFILE_NAME)));
         }
+        
+        AuthConfig buildAuthConfig = new AuthConfig()
```

Comment:
It'll use default docker hub and your logon name if values are null, but probably still a good idea to only set the auth config in case username and password are not null